src/index.html

```diff
@@ -4,9 +4,443 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#Home" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo image & Link to Home page">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photos" class="nav__link">
+              Фото
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Видео
+            </a>
+          </li>
+
+        </ul>
+      </nav>
```

Comment:
no empty line between parent and child tag

```suggestion
          </li>
        </ul>
      </nav>
```